hax/hax/motr/hax.c

```diff
@@ -276,13 +276,13 @@ int m0_ha_pver_status(unsigned long long ctx,
 	struct hax_context *hc = (struct hax_context *)ctx;
 	struct m0_halon_interface *hi = hc->hc_hi;
 	struct m0_spiel *spiel = m0_halon_interface_spiel(hi);
-	struct m0_confc *confc = spiel->spl_core.spc_confc;
+
 	struct m0_conf_pver_info pver_info;
 
 	int rc;
 	// call the motr api
```

Comment:
No c99 style comments please. Please use `/* */`, at other places as well.

Reply:
done

Reply:
@Shreya-18 There are couple of c99 style comments present in same file(not added as part of this PR though).
Lets remove them in PR as discussed.